base/src/y2base.cc

```diff
@@ -1,22 +0,0 @@
-/*---------------------------------------------------------------------\
-|                                                                      |
-|                      __   __    ____ _____ ____                      |
-|                      \ \ / /_ _/ ___|_   _|___ \                     |
-|                       \ V / _` \___ \ | |   __) |                    |
-|                        | | (_| |___) || |  / __/                     |
-|                        |_|\__,_|____/ |_| |_____|                    |
-|                                                                      |
-|                               core system                            |
-|                                                        (C) SuSE GmbH |
-\----------------------------------------------------------------------/
-
-   File:       y2base.cc
-
-   Author:     Mathias Kettner <kettner@suse.de>
-   Maintainer: Mathias Kettner <kettner@suse.de>
-
-/-*/
-/*
- * Hi. This file is empty on purpose :-)
- * Nice, isn't it?
```

Comment:
well, in past it was used also for scr binary. But we probably also no longer use it. So it maybe also make sense to drop it.